meteomatics/rounding.py

```diff
@@ -1,47 +1,59 @@
 # -*- coding: utf-8 -*-
 
 decimal_places = {
-    "cm":       1,  # centimeters (snow height)
+    "bft":      0,  # Beauforts (wind)
+    "cm":      -1,  # centimeters (snow height, hail)
     "C":        1,  # temperature
     "d":       -1,  # days AND degrees...
     "dn":       5,  # datenums
-    "eur":      1,  # euro
     "F":        1,  # temperature
     "ft":       1,  # feet (ceiling height)
+    "gcm2":     4,  # total column air pollution
```

Comment:
is that not a bit much? Where do we have that unit?

Reply:
The total column parameters for example: tc_ch4:gcm2